### PR TITLE
Fix test events filter container

### PR DESCRIPTION
**Pull request by DarianAnjuhal**
Fixed pretty simple issue:
https://github.com/docker/docker/issues/12660

I needed time to understand whats going on with compiling, testing and the go language (which is weird).

### CODE REVIEW

**Comment by DarianAnjuhal** [comment]
I did the signature stuff already, should it automatically update or did I an error?

**Comment by duglin** [comment]
You need to sign each commit, so squashing it down to one will fix it.

**Comment by LK4D4** [comment]
Lol, thanks. Also you need to check error from wait inspect.

File: integration-cli/docker_cli_events_test.go [hunk 1]
```diff
@@ -327,7 +327,7 @@ func (s *DockerSuite) TestEventsFilterContainer(c *check.C) {
 			c.Fatal(err)
 		}
 		nameID[name] = strings.TrimSpace(out)
-		waitInspect(name, "{{.State.Runing }}", "false", 5)
+		waitInspect(name, "{{.State.Running }}", "false", 5)
```
Comment (icecrime) [inline integration-cli/docker_cli_events_test.go]: Can we please replace this by a call to [`waitRun`](https://github.com/docker/docker/blob/eae272f90eb6c01824112054140b5dc7d9d679ce/integration-cli/utils.go#L202-L204)?

**Comment by DarianAnjuhal** [comment]
He he, I knew this would be to easy. :-)

File: integration-cli/docker_cli_events_test.go (hunk 1 above)
Comment (DarianAnjuhal) [inline integration-cli/docker_cli_events_test.go]: Okay, I will change it.

**Comment by icecrime** [comment]
@DarianAnjuhal It's not that easy, but don't worry it shouldn't be too hard ;-)
* [Squash your commits](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html)
* Replace the call to `waitInspect` by [`waitRun`](https://github.com/docker/docker/blob/eae272f90eb6c01824112054140b5dc7d9d679ce/integration-cli/utils.go#L202-L204)
* Verify `waitRun` return value as suggested by @LK4D4 
* Profit! (well, not really, this is open source, but thanks!)

**Comment by DarianAnjuhal** [comment]
Weired, there is the Signed-off-by: ... stuff in the comment and it still doesnt work?

**Pull request closed without merging**